README.rst

```diff
@@ -92,8 +92,7 @@ How?
 
 1. Get a personal access token with permission to manage your *gists*
    from `GitHub settings <https://github.com/settings/tokens>`__
-2. Set an environment variable called ``GETGIST_TOKEN`` with your
-   personal access token
+2. Set an `environment variable <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ called                  ``GETGIST_TOKEN`` with your personal access token
```

Comment:
Many thanks, @kmmelcher : )

Here is one os my preferred conundrums of documenting open-source packages: whether we should everything or limit the documentation to what the package actually does.

In this specific case, creating environmental variables is not something directly related to this package, although it is still important for a nicer UX in this package. Then, thinking about that, we decide to document it. However, now we have to think about Windows, macOS, and GNU/Linux users. And to consider they might be using Bash, Zsh, or PowerShell… and then, to cover that all, we ended up writing more and more documentation that is not related to this package — our task ends up being teaching users how to use their OS and tooling and not to teach them how to use GetGist. And we are back to step 1.

Long story short, we just need to be clear about where are the boundaries. What do you think about slightly changing the wording and formatting? I suggest keeping this line as it is and adding another one like:

> This [article](https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux) might help you create an environment variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.